testsuite/Makefile

```diff
@@ -335,6 +335,10 @@ clean:
 	find . -name '*_ocamltest*' | xargs rm -rf
 	rm -f $(failstamp)
 
+.PHONY: distclean
+distclean: clean
+	rm -f _log*
```

Comment:
This needs to do `$(MAKE) -C tools distclean` and `$(MAKE) -C lib distclean`, I think?